package.json

```diff
@@ -33,21 +33,29 @@
     "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
+    "@fluffy-spoon/substitute": "^1.107.0",
     "@types/body-parser": "^1.16.5",
+    "@types/chai": "^4.2.8",
     "@types/cookie-parser": "^1.4.2",
     "@types/dotenv": "^8.2.0",
     "@types/express": "^4.0.37",
     "@types/express-session": "^1.15.3",
+    "@types/mocha": "^7.0.1",
     "@types/nunjucks": "^3.1.3",
     "@types/redis": "^2.8.14",
+    "@types/supertest": "^2.0.8",
+    "chai": "^4.2.0",
     "del": "^5.1.0",
     "gulp": "^4.0.2",
     "gulp-nodemon": "^2.4.2",
     "gulp-sass": "^4.0.2",
     "gulp-typescript": "^6.0.0-alpha.1",
     "gulp-watch-sass": "^1.4.1",
+    "mocha": "^7.0.1",
     "node-sass": "^4.13.1",
     "sass-lint": "^1.13.1",
+    "supertest": "^4.0.2",
```

Comment:
https://github.com/visionmedia/supertest - allows spinning up instance of application with selected route registered and testing it via series of HTTP calls. It will allow us test integration between application framework, DI container and route implementation.